### PR TITLE
LPS-186204 Add possibility to associate Accounts with Organizations

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java [hunk 1]
```diff
@@ -455,6 +456,17 @@ private void _assertAccountGroups2() throws Exception {
...
 
+	private void _assertAccountOrganizationRelsCount(
+			Organization organization, int organizationAssignmentsCount)
+		throws Exception {
+
+		Assert.assertEquals(
+			organizationAssignmentsCount,
+			_accountEntryOrganizationRelLocalService.
+				getAccountEntryOrganizationRelsByOrganizationIdCount(
+					GetterUtil.getLong(organization.getId())));
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java:467]: do you really need to use GetterUtil?

**Review by nNilton — CHANGES_REQUESTED** [review]

**Review by nNilton — APPROVED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:foward

**Comment by joaodamasceno2001** [comment]
ci:forward

**Pull request closed without merging**